provision/core/mailhog/provision.sh

```diff
@@ -45,22 +45,24 @@ function mailhog_setup() {
 
   # Start on reboot
   if [ "${VVV_DOCKER}" != 1 ]; then
+    # no systemd
     vvv_info " * Enabling MailHog Service"
-    systemctl enable mailhog
+    # systemctl enable mailhog
 
     vvv_info " * Starting MailHog Service"
-    systemctl start mailhog
+    # systemctl start mailhog
+    /usr/bin/env /usr/local/bin/mailhog > /dev/null 2>&1 &
+    vvv_info " * Started MailHog Service"
   fi
 }
 export -f mailhog_setup
 
 vvv_add_hook after_packages mailhog_setup
 
 function mailhog_restart() {
-  if [ "${VVV_DOCKER}" != 1 ]; then
-    service mailhog restart
-  fi
+  killall mailhog; /usr/bin/env /usr/local/bin/mailhog > /dev/null 2>&1 &
```

Comment:
what happens if you run `sudo service mailhog restart` on the docker VM?

Reply:
at the moment it says`mailhog: unrecognized service`
```
vagrant@vvv:~$ sudo service mailhog restart
mailhog: unrecognized service
vagrant@vvv:~$
```